### DISCUSSION TITLE
How to use --include-cluster-scoped-resources?

### DISCUSSION THREAD

**Discussion by Muerti** [vmware-tanzu/velero#7781] [Community support Q&A] [answered]
Unclear using of such options. 
What i have tried:
1)use only includedClusterScopedResources = '*'
Result: backup all resources
2) use includedClusterScopedResources с перечислением ресурсов
Result: backuping all namespaced resources and skipping includedClusterScopedResources.

**Comment by kaovilai** [comment]
https://velero.io/docs/v1.13/resource-filtering/

**Reply by Muerti** [reply, marked as answer]
Thank you for your replay. 
Yes, I have read doc before asking question.
The problem was with format of resource kind. Unfortunately, valid formats are not documented well.

Sorry for my English. Best regards.
